scripts/dockerfiles/polkadot/build.sh

```diff
@@ -8,17 +8,20 @@ PROJECT_ROOT=`git rev-parse --show-toplevel`
 cd $PROJECT_ROOT
 
 # Find the current version from Cargo.toml
-VERSION=`grep "^version" ./cli/Cargo.toml | egrep -o "([0-9\.]+)"`
+VERSION=`grep "^version" ./cli/Cargo.toml | egrep -o "([0-9\.]+-?[0-9]+)"`
```

Comment:
i would do something like this instead:
```suggestion
VERSION=`grep "^version" ./cli/Cargo.toml | egrep -o "([0-9\.-]+)"`
```

Reply:
Why that ? Do you expect some `0.9-1.13` versions or something looking like that ?

Reply:
🤷‍♀️  Not necessarily, but shorter regexes are nicer for maintainability IMO.
Fine with either, really